pkg/meta/types.go

```diff
@@ -68,10 +68,10 @@ type DumpedAttr struct {
 
 type DumpedSlice struct {
 	Pos     uint32 `json:"pos"`
```

Comment:
How about move pos to the end? We'd like to have chunkid and Size together for readability.

Reply:
Yes, we can place it to be between Len and Chunkid.

Reply:
Or we can just swap these two lines so that they stay together.